tests/log-in.test.js

```diff
@@ -0,0 +1,75 @@
+//const { default: $ } = require("webdriverio/build/commands/browser/$");
+describe('Banking', () => {
+
+    it('Debería ingresar a la pagina con datos correctos', async () => {
+        await browser.url('login');
+
+        let userName = await $('#username');
+        await userName.setValue('jsmith@demo.io');
+
+        let pssWord = await $('#password');
```

Comment:
prefer complete name, as they're more readable (just `let password`). Also, password is a single word, so no need to case the W.